protocol-definitions/MC.yaml

```diff
@@ -859,3 +859,37 @@ methods:
          since: 2.4
          doc: Name of the Queue for which the age-related statistics should be reset to 0
     response: {}
+  - id: 34
+    name: reloadConfig
+    since: 2.4
+    doc: Reloads the configuration on the member from its config file, applies (merges) the new config to the current config
+    request:
+      retryable: false
+      partitionIdentifier: -1
+    response:
+      params:
+        - name: configUpdateResult
```

Comment:
we need to get information about the error in case of configuration update failure

Reply:
If we make this async (as per your other comment), then the failure reason will be included in the `ConfigUpdateFailedEvent` we receive on the MC side, won't it?

Reply:
right

Reply:
ok thanks I will rework the PRs to make this operation async